app/javascript/components/enrollment/steps/Exposure.js

```diff
@@ -102,10 +133,71 @@ class Exposure extends React.Component {
     );
   }
 
+  updateLDEandCEValidations = patient => {
+    if (!patient.last_date_of_exposure && !patient.continuous_exposure) {
+      schema = yup.object().shape({
+        ...staticValidations,
+        last_date_of_exposure: yup
+          .date('Date must correspond to the "mm/dd/yyyy" format.')
+          .max(
+            moment()
+              .add(30, 'days')
+              .toDate(),
+            'Date can not be more than 30 days in the future.'
+          )
+          .required('Please enter a Last Date of Exposure OR turn on Continuous Exposure')
+          .nullable(),
+        continuous_exposure: yup.bool().nullable(),
+      });
+    } else if (!patient.last_date_of_exposure && patient.continuous_exposure) {
+      schema = yup.object().shape({
+        ...staticValidations,
+        last_date_of_exposure: yup
+          .date('Date must correspond to the "mm/dd/yyyy" format.')
+          .max(
+            moment()
+              .add(30, 'days')
+              .toDate(),
+            'Date can not be more than 30 days in the future.'
+          )
+          .oneOf([null, undefined])
+          .nullable(),
+        continuous_exposure: yup
+          .bool()
+          .oneOf([true])
+          .nullable(),
+      });
+    } else if (patient.last_date_of_exposure && !patient.continuous_exposure) {
+      schema = yup.object().shape({
+        ...staticValidations,
+        last_date_of_exposure: yup
+          .date('Date must correspond to the "mm/dd/yyyy" format.')
+          .max(
+            moment()
+              .add(30, 'days')
+              .toDate(),
+            'Date can not be more than 30 days in the future.'
+          )
+          .required()
+          .nullable(),
+        continuous_exposure: yup
+          .bool()
+          .oneOf([null, undefined, false])
+          .nullable(),
+      });
```

Comment:
so I originally had that case in there, but since LDE automatically gets cleared out when CE is checked, that case is not even possible, so I removed it because it was unnecessary code that would never be hit

Reply:
ah for the existing records, I agree that it wouldn't hurt to have it in here. I'll add that then

Reply:
addressed here: https://github.com/SaraAlert/SaraAlert/pull/418/files